package.json

```diff
@@ -240,73 +240,74 @@
-		"test-php": "npm run lint-php && npm run test-unit-php",
-		"test-php:watch": "wp-env run composer run-script test:watch",
-		"test-unit": "wp-scripts test-unit-js --config test/unit/jest.config.js",
-		"test-unit:date": "bash ./bin/unit-test-date.sh",
-		"test-unit:debug": "wp-scripts --inspect-brk test-unit-js --runInBand --no-cache --verbose --config test/unit/jest.config.js ",
-		"test-unit:update": "npm run test-unit -- --updateSnapshot",
-		"test-unit:watch": "npm run test-unit -- --watch",
-		"pretest-unit-php": "wp-env start",
-		"test-unit-php": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose",
-		"pretest-unit-php-multisite": "wp-env start",
-		"test-unit-php-multisite": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit/multisite.xml --verbose",
+		"publish:patch": "lerna publish patch --no-private --dist-tag patch",
+		"start": "npm run dev",
 		"prestorybook:build": "npm run build:packages",
 		"storybook:build": "build-storybook -c ./storybook -o ./storybook/build",
 		"prestorybook:dev": "npm run build:packages",
 		"storybook:dev": "concurrently \"npm run dev:packages\" \"start-storybook -c ./storybook -p 50240\"",
-		"env": "wp-scripts env",
-		"wp-env": "wp-env",
-		"prepare": "husky install"
+		"test": "npm-run-all lint test:unit",
+		"test:create-block": "bash ./bin/test-create-block.sh",
+		"test:e2e": "wp-scripts test:e2e --config packages/e2e-tests/jest.config.js",
+		"test:e2e:debug": "wp-scripts --inspect-brk test-e2e --config packages/e2e-tests/jest.config.js --puppeteer-devtools",
+		"test:e2e:playwright": "playwright test --config test/e2e/playwright.config.ts",
+		"test:e2e:watch": "npm run test:e2e -- --watch",
+		"test:performance": "wp-scripts test-e2e --config packages/e2e-tests/jest.performance.config.js",
+		"test:php": "npm-run-all lint:php test:unit:php",
+		"test:php:watch": "wp-env run composer run-script test:watch",
+		"test:unit": "wp-scripts test-unit-js --config test/unit/jest.config.js",
+		"test:unit:date": "bash ./bin/unit-test-date.sh",
+		"test:unit:debug": "wp-scripts --inspect-brk test-unit-js --runInBand --no-cache --verbose --config test/unit/jest.config.js ",
+		"pretest:unit:php": "wp-env start",
+		"test:unit:php": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose",
+		"pretest:unit:php:multisite": "wp-env start",
+		"test:unit:php:multisite": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit/multisite.xml --verbose",
+		"test:unit:update": "npm run test:unit -- --updateSnapshot",
+		"test:unit:watch": "npm run test:unit -- --watch",
+		"wp-env": "wp-env"
```

Comment:
This one can be removed since we have `npm run env` and you can always run `npx wp-env` that is shorter and doesn't require additional `--` to pass params.

Reply:
Fixed with e69de76👍